.github/pull_request_template.md

```
@@ -1,27 +1,45 @@
-## What changes are proposed in this pull request?
+## 🔗 Related Issues
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)